frame/babe/src/lib.rs

```diff
@@ -137,6 +139,11 @@ pub mod pallet {
 		/// when no other module is responsible for changing authority set.
 		type EpochChangeTrigger: EpochChangeTrigger;
 
+		/// A way to check whether a given validator is disabled and should not be authoring blocks.
+		/// Blocks authored by a disabled validator will lead to a panic as part of this module's
+		/// initialization.
+		type DisabledValidators: DisabledValidators;
```

Comment:
note: because babe implements SessionHandler, it should already get this information when `SessionHandler::on_disabled` is triggered. But I agree this implementation allow not to store this information in this pallet and instead only store it in the session pallet.

So I'm ok with this implementation, although duplicating the storage would make the configuration more easy.

Reply:
Yeah, I agree. I have another PR open where I needed to duplicate this data to the staking module, so it felt a bit bad to keep duplicating this further (although it in practice this should always be empty...)